Dockerfile

```diff
@@ -14,7 +14,8 @@ COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
 COPY pkg/ pkg/
-COPY bindata/ bindata/
+COPY bindata/deployment/ bindata/deployment/
```

Comment:
https://github.com/metallb/metallb-operator/pull/8
introduces a script that relies on bindata/
putting this comment so we'll know about it when one of the PRs merges before the other

Reply:
sounds good Thanks @oribon